resources/building-mobile-extensions/get-the-application-context.md

```diff
@@ -0,0 +1,36 @@
+# Get the Application Context
+
+In Android, you can retrieve the `android.app.Application` instance using the `MobileCore.getApplication()` API. The returned `android.app.Application` represents the base class which maintains the global application state. For more information, see [Moble Core API Reference](../../using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md)
```

Comment:
- In the 2nd sentence, please add a comma between "class" and "which".
- Please add a period at the end of the 3rd sentence.